sdk/eventhub/azure-eventhub/samples/sync_samples/authenticate_with_named_key_credential.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to demonstrate utilizing AzureNamedKeyCredential to authenticate with Event Hubs.
+"""
+
+# pylint: disable=C0111
+
+import os
+import time
+from azure.core.credentials import AzureNamedKeyCredential
```

Comment:
Is this the same as importing `AzureNamedKeyTokenCredential` from `eventhub` and using that to authenticate?

Reply:
same answer as above :)